### PR TITLE
More current no sets found copy.

### PR DESCRIPTION
**Pull request by ryan-rushton**
Screen Shot 2021-07-17 at 2 05 12 pm

### CODE REVIEW

**Comment by nev-r** [comment]
"cause by the inability to assign" is a bit labyrinthine, even in english.
but especially for i18n, let's keep things to like, ELI5 style words & structure.

can i suggest something like
"No builds were found. This is typically because DIM could not assign the selected mods to the selected armor."

**Comment by ryan-rushton** [comment]
Sounds good, I tweaked what you had slightly but same gist.
